app/logic/users.py

```diff
@@ -26,7 +26,7 @@ def addUserInterest(program_id, username):
     username: username of the user showing interest
     """
     Interest.get_or_create(program = program_id, user = username)
-    return True
+    return ""
```

Comment:
These functions are supposed to return True if they succeed. The error is in what comes back from the controller.